crates/codegen/parser_runtime/src/support/parser_result.rs

```diff
@@ -68,6 +123,18 @@ impl IncompleteMatch {
             }
         }
     }
+
+    pub fn is_better_match_than(&self, other: &IncompleteMatch) -> bool {
+        let first_size = self
+            .nodes
+            .iter()
+            .fold(0, |acc, node| acc + node.text_len().utf8);
+        let second_size = other
+            .nodes
+            .iter()
+            .fold(0, |acc, node| acc + node.text_len().utf8);
+        first_size < second_size
```

Comment:
What's the criteria for a "better match"? If *matches more text*, then shouldn't this be `>` instead (i.e. we cover more text than the `other`)?

Reply:
good catch!